pkg/api/types.go

```diff
@@ -38,6 +38,9 @@ type DeschedulerPolicy struct {
 
 	// MaxNoOfPodsToEvictPerNamespace restricts maximum of pods to be evicted per namespace.
 	MaxNoOfPodsToEvictPerNamespace *uint
+
+	// MinNoOfPodsToEvictForReplicas restricts minimal of pod's ownerReference to avoid evicting.
+	MinNoOfPodsToEvictForReplicas *uint
```

Comment:
This option is more suitable for the `filter` extension point of the `DefautlEvictor`. So it can be used for categorizing pods are non-evictable during balancing as well.

Reply:
I think it is a great idea. I will refine code next.